implement get started section (#8107)
getsentry/sentry-docs: src/docs/product/sentry-basics/index.mdx

---
title: Get Started with Sentry
sidebar_order: 10
description: "These tutorials will get you started with using Sentry."
redirect_from:
  - /basics/
  - /product/sentry-basics/guides/
---

> ## What is Sentry?
> 
> Sentry is a software monitoring tool that helps developers identify and fix code-related issues. From error tracking to performance monitoring, Sentry provides code-level observability that makes it easy to diagnose issues and learn continuously about your application code health.

Reviewer: ```suggestion
## What is Sentry?

Sentry is a software monitoring tool that helps developers identify and fix code-related issues. From error tracking to performance monitoring, Sentry provides code-level observability that makes it easy to diagnose issues and learn continuously about your application code health.

## How to Get the Most Out of Sentry

There are three things we recommend you do to get the most out of Sentry:

- **Connect your codebase**, being sure you've also set up [source code management integrations](/product/integrations/source-code-mgmt/) and [source maps](/product/sentry-basics/integrate-frontend/upload-source-maps/) (if applicable). This will allow Sentry to link errors and performance issues directly to the problematic line of code via readable stack traces and suspect commits.
- **Set up [alerts](/product/alerts/)** to make sure you know when things you care about are going wrong. Be sure to follow [alert best practices](/product/alerts/best-practices/), so that you only get notified about things you care about.
- **Customize [Discover](/product/discover-queries/), [Dashboards](/product/dashboards/), and [queries](/product/discover-queries/query-builder/)** to unlock custom insights about the things that are going wrong in your app. 
```

I think this is a REALLY important section and should live at the top.

- [Issue tracking](/product/integrations/issue-tracking/) with Jira, Github, and many other tools
- [Alerts and notifications](/product/integrations/notification-incidents/) with Slack, Discord, PagerDuty, and more

### SDK Support for Over 100 Platforms

Sentry supports over [100 platforms and frameworks](/platforms/), including 30+ coding languages, so you can track errors and performance issues no matter what your application is built on. Our SDKs make it easy to get started with Sentry — it only takes a few lines of code!

## Get Started Now

If you haven't already, sign up for Sentry to create a new Sentry organization and user account.

**→ [Sign up for Sentry](https://sentry.io/signup/)**

Once you have an account, you can create a new Sentry project, choosing from dozens of frameworks. Copy the initialization method into your app to add the Sentry SDK, and Sentry will start sending errors and performance events to your new project.

**→ [Set up error monitoring for a frontend project](/product/sentry-basics/integrate-frontend/)**

> After you have Sentry set up, make sure to do these 3 things to help you get the most value out of Sentry:
> 
> - Set up [source code management integrations](/product/integrations/source-code-mgmt/) and [source maps](/product/sentry-basics/integrate-frontend/upload-source-maps/) (if applicable) so Sentry can link errors and performance issues directly to the problem line of code via readable stack traces and suspect commits.
> - Set up [alerts](/product/alerts/), following [best practices](/product/alerts/best-practices/), so that you only get notified for things you care about.
> - Use [Discover](/product/discover-queries/), [Dashboards](/product/dashboards/), and [queries](/product/discover-queries/query-builder/) to unlock custom insights. These tools allow you to navigate through error and performance data across your entire system and create custom queries and widgets to uncover trends and learn more about your app.
> 

Reviewer: ```suggestion

```
I think we should move this section up to the top.

> ### Tutorials and more

Reviewer: ```suggestion
### Tutorials and Concepts
```

> Use the tutorials and resources below to help you get started with Sentry.

Reviewer: ```suggestion
Get insights and fix what's broken with help from the below tutorials and concepts. 
```
